packages/gsm8k/README.md

```diff
@@ -28,5 +27,5 @@ R. and Hesse, C., 2021.
 To install the GSM8k environment, run the following command:
 
 ```bash
-pip install fhaviary[gsm8k]
+pip install 'fhaviary[gsm8k]'
```

Comment:
Fwiw we have `bash` syntax here, so `'` isn't necessary.

`'` is only necessary for `zsh`

Reply:
Feel free to ignore this, I am just pointing out is all

Reply:
I tried running it in zsh and got an error because I was unaware of the syntax. Let me use `pip install fhaviary[gsm8k]` and add a comment about zsh for non-engineers

Reply:
Resolved offline